Author: Simon Frankenberger <simon-alpine@fraho.eu>
Subject: Fix compilation of jtreg / gtest objects with -Werror=format-security CFLAGS
Upstream: Yes / closed unresolved (https://bugs.openjdk.org/browse/JDK-8218814)

Remove the -Wno-format CLFAG for test objects

diff -ru old/make/common/TestFilesCompilation.gmk new/make/common/TestFilesCompilation.gmk
--- old/make/common/TestFilesCompilation.gmk
+++ new/make/common/TestFilesCompilation.gmk
@@ -112,7 +112,7 @@
         CXXFLAGS := $$(TEST_CFLAGS) $$($1_CFLAGS) $$($1_CFLAGS_$$(name)), \
         LD_SET_ORIGIN := $$($1_LD_SET_ORIGIN), \
         LDFLAGS := $$($1_LDFLAGS_$$(name)), \
-        DISABLED_WARNINGS_gcc := format undef unused-but-set-variable \
+        DISABLED_WARNINGS_gcc := undef unused-but-set-variable \
             unused-const-variable unused-function unused-value \
             unused-variable, \
         DISABLED_WARNINGS_clang := format-nonliteral \
